Source/AssetRipper.GUI.Electron/Pages/Assets/View.cshtml.cs

```diff
@@ -60,6 +62,34 @@ static DirectBitmap TextureToBitmap(ITexture2D texture)
 			}
 		}
 
+		public byte[]? Font
+		{
+			get
+			{
+				return Asset is IFont font ? font.FontData_C128 : null;
+			}
+		}
+
+		public string? FontType
+		{
+			get
+			{
+				if (Font is { Length: >= 4 })
+				{
+					return (Font[0], Font[1], Font[2], Font[3]) switch
+					{
+						(0x4F, 0x54, 0x54, 0x4F) => "font/otf",    // "OTTO" in ASCII
+						(0x77, 0x4F, 0x46, 0x46) => "font/woff",   // "wOFF" in ASCII
+						(0x77, 0x4F, 0x46, 0x32) => "font/woff2",  // "wOF2" in ASCII
+						(0x00, 0x01, 0x00, 0x00) => "font/ttf",    // 0x00010000 as uint32
```

Comment:
Put this above the two woff formats. I don't think woff is ever used in Unity, so it's inefficient to check them first.